finish/src/test/java/it/io/openliberty/guides/config/ConfigTestUtil.java

```diff
@@ -111,9 +115,15 @@ public static void overwriteFile(String fileName, String newContent) {
    * Get the Json Object from the URL provided.
    */
   public static JsonObject getJsonObjectFromURL(Client client, String url, int level,
-      String key) {
+      String key, Optional<Integer> expectedStatus) {
     Response response = client.target(url).request().get();
 
+    if (expectedStatus.isPresent()) {
+      Integer expected = expectedStatus.get();
+      Integer actual = response.getStatus();
+      assertEquals(expected, actual);
```

Comment:
Generally we want to have the assert statement in the actual test case class, so it's easier to tell what are being compared.